src/style.css

```diff
@@ -0,0 +1,39 @@
+@import "./styles/card.css";
+@import "./styles/stars.css";
+@import "./styles/root.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.button {
+  font-size: 14px;
+  font-weight: 700;
+  text-transform: uppercase;
+  padding: 11px 0;
+  color: #fff;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: all 0.2s linear;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated